### PR TITLE
Change install script to use prebuilt xenial .debs

### PR DESCRIPTION
**Pull request by jonlawlor**
This circumvents the typical ubuntu apt-get and installs blas and lapack libraries from ubuntu xenial packages. Followup to gonum/blas#170.

### CODE REVIEW

**Comment by jonlawlor** [comment]
PTAL @kortschak

File: .travis/linux/OpenBLAS/install.sh [hunk 1]
```diff
@@ -1,28 +1,31 @@
...
-sudo apt-get update -qq && sudo apt-get install -qq gfortran
+# remove exiting blas packages to prevent conflicts
+sudo apt-get remove libblas3gf libatlas3gf-base
 
-# fetch OpenBLAS
-pushd ~
-sudo git clone --depth=1 git://github.com/xianyi/OpenBLAS
+# fetch fortran to build OpenBLAS
+sudo apt-get update -qq
+sudo apt-get install -qq gfortran
```
Comment (kortschak) [inline .travis/linux/OpenBLAS/install.sh]: Why is this needed?

**Comment by kortschak** [comment]
Same comment. This is pretty fragile.

**Comment by vladimir-ch** [comment]
> Same comment. This is pretty fragile.

I've been thinking the same.

File: .travis/linux/OpenBLAS/install.sh (hunk 1 above)
Comment (jonlawlor) [inline .travis/linux/OpenBLAS/install.sh]: From gonum/lapack#115
> Yes, libblas3gf in Ubuntu 12.04 has a dependency on libblas-common, but the dependency tree is different than in Ubuntu Xenial, so it results in a circular dependency.

**Comment by jonlawlor** [comment]
From gonum/lapack#115
>Yes, it is definitely fragile. If we want faster builds then we need prebuilt packages (I think, although there may be some make options in OpenBLAS that can reduce compilation time...). Travis does not allow manipulation of sources.list, so we have to download and install packages outside of apt-get. It may be possible to get the packages from somewhere else, and we could make it a bit better by having multiple hosts that can provide the .deb files, and we can pretty it up a bit, but it will never be as stable as apt-get, and we lose the benefit of compiling against the latest OpenBLAS to see if they have any regressions. It is up to you guys if the tradeoff is worth it.

**Comment by jonlawlor** [comment]
Closed as per gonum/lapack#172, unless anyone feels very strongly about ci times.
